graylog2-server/src/main/java/org/graylog2/Configuration.java

```diff
@@ -159,8 +158,9 @@
     @Parameter(value = "deactivated_builtin_authentication_providers", converter = StringSetConverter.class)
     private Set<String> deactivatedBuiltinAuthenticationProviders = Collections.emptySet();
 
+    // This is needed for backwards compatibility. The setting in TLSProtocolsConfiguration should be used instead.
     @Parameter(value = "enabled_tls_protocols", converter = StringSetConverter.class)
-    private Set<String> enabledTlsProtocols = DefaultTLSProtocolProvider.getDefaultSupportedTlsProtocols();
+    public Set<String> enabledTlsProtocols = null;
```

Comment:
Can we make this private again and add a getter with a `@Deprecated` annotation? That makes it clear in the IDE that the usage is deprecated. I think adding the annotation to the public field doesn't highlight it in the IDE.

Reply:
I thought not adding a getter would send the same message.. 
But sure I could do that